packages/fxa-settings/src/components/Settings/Tooltip/index.stories.tsx

```diff
@@ -53,7 +53,7 @@ storiesOf('Components/Settings/Tooltip', module)
           <Tooltip
             message="My tooltip message here"
             position="bottom"
-            anchorLeft={true}
+            anchorStart={true}
```

Comment:
More of a btw and not even a nit, it's totally fine to explicitly set `true` for `true` props and I know some folks prefer it but you can also simply pass in `anchorStart` instead of `anchorStart={true}` for some syntactic sugar.